pkg/sgtm/page_post.go

```diff
@@ -205,7 +174,9 @@ func (svc *Service) postMaintenancePage(box *packr.Box) func(w http.ResponseWrit
 				return
 			}
 			svc.logger.Debug("BPM extracted", zap.Float64("bpm", bpm))
-			if err := svc.rwdb().Model(&post).Update("bpm", bpm).Error; err != nil {
+			post.BPM = bpm
+			err = svc.storage.GenericUpdatePost(&sgtmpb.Post{}, post.BPM)
```

Comment:
```suggestion
			err = svc.storage.GenericUpdatePost(post, &sgtmpb.Post{"BPM", post.BPM})
```

With this, we've the same behavior than before